website/docs/core-concepts/data-access-and-binding/capturing-data-write/capture-form-data.md

```diff
@@ -33,6 +40,49 @@ Once the API / Query updates the data, the widgets on the screen need to be upda
 The Property Pane UI supports a single onSuccess Callback but multiple callbacks and conditions can be configured by clicking the JS button next to the property. Learn more about [Creating Workflows](../../writing-code/workflows.md)
 :::
 
-## Posting URL Encoded Form Data
+## Send body data with API requests
```

Comment:
@spark-c I think there is some confusion. This section will be a How to guide which is why all of review comments were made on that page itself. The was to move URL-encoded form data to that page. I think because of the confusion everything is moved here. Can you quickly fix that. This page talks about Capturing data entered in the UI.  We can cross-reference the information here but the data should reside in How to guides